includes/blogdb.sql

```diff
@@ -0,0 +1,139 @@
+-- phpMyAdmin SQL Dump
```

Comment:
This File doesn't seem to be a part of this issue. Can we remove this file from this PR. and create a separate one for including this database file?

Reply:
@vbadkar ^